jackbox_patcher/lib/components/closable_route_with_esc.dart

```diff
@@ -27,20 +29,28 @@ class ClosableRouteWithEsc extends StatefulWidget {
 }
 
 class _ClosableRouteWithEscState extends State<ClosableRouteWithEsc> {
+  bool _handleNavigateBack(BuildContext context) {
+    if (Navigator.of(context).canPop()) {
+      if (widget.stopVideo) {
+        VideoService.stop();
+      }
+      if (widget.closeSFX) {
+        SFXService().playSFX(SFX.CLOSE_GAME_INFO_TAB);
+      }
+      Navigator.of(context).pop();
+      return true;
+    }
+    return false;
+  }
+
   @override
   Widget build(BuildContext context) {
     return Focus(
       autofocus: true,
       onKey: (node, event) {
-        if (event.isKeyPressed(LogicalKeyboardKey.escape)) {
-          if (Navigator.of(context).canPop()) {
-            if (widget.stopVideo) {
-              VideoService.stop();
-            }
-            if (widget.closeSFX) {
-              SFXService().playSFX(SFX.CLOSE_GAME_INFO_TAB);
-            }
-            Navigator.of(context).pop();
+        if (event.isKeyPressed(LogicalKeyboardKey.escape) ||
+            event.isKeyPressed(LogicalKeyboardKey.backspace)) {
+          if (_handleNavigateBack(context)) {
```

Comment:
We cant simply return _handleNavigateBack, we only return something if its true else we dont return anything